README.md

```diff
@@ -151,10 +150,12 @@ Returns a `Promise` that resolves to an object with the entry's properties eg:
 ```js
 {
   value: Uint8Array,
-  signature: Uint8Array,
+  signature: Uint8Array, // V1 (legacy, ignored)
```

Comment:
link to this PR and https://github.com/ipfs/js-ipfs/pull/4207?

Reply:
imo unnecessary noise for readme – is someone wants to dig, they will be able to read specs: https://github.com/ipfs/specs/pull/319